functions/tips-lazy-globals/main.py

```diff
@@ -28,13 +28,17 @@ def function_specific_computation():
 
 # [START functions_tips_lazy_globals]
 # [START cloudrun_tips_global_lazy]
+import functions_framework
```

Comment:
To fix the lint error:
1) Move this import statement to the top of the file (e.g. line 15)
2) Break the sample up into multiple region tags, a la:
```
// [START tag]
<imports go here>
// [END tag]

...

// [START tag]
<code goes here>
// [END tag]
```